### DISCUSSION TITLE
A nice way to access packets

### DISCUSSION THREAD

**Discussion by AlmanaX-21** [PurpurMC/Purpur#973] [__Dismissed__] [answered]
I think packets need a better way to be accessed for developers, if purpur adds easy to way to send and receive packets in-built into the developer api that in my opinion would be game changing

**Comment by BillyGalbreath** [comment, marked as answer]
If you use [paperweight's userdev bundle](https://github.com/PaperMC/paperweight-test-plugin) for your project, you will get full access to deobfuscated NMS which makes using things like packets in a plugin super simple. There is no need for such an advanced API anymore, built into the server or a 3rd party like ProtocolLib. Just use NMS if you need something that advanced. ;)
